### ISSUE TITLE
Parameter "Sequence Length" for the Model

### ISSUE THREAD

**Issue by HollowFire** [pytorch/fairseq#1481]
Hi, I'm trying to reproduce the experimental results in the paper "Dynamic Convolutions" (Wu et al., 2019) by referencing this repo but using tensorflow. I have built the basic models but encountered a problem when preprocessing the dataset (iwlst 14 de-en). I can't find such a paramter as "src_len" or "tgt_len" in the preprocess script. This parameter is not mentioned in the paper either.

To my understanding, seq2seq tasks require setting a maximum sequence length, padding shorter sentences to this length and removing sentences that are longer. I'm sorry if my quenstion sounds stupid, but anyone can give me a clue? Thank you.

**Comment by lematt1991** [comment]
This isn't done in preprocessing, its done on the fly.  You can set the [`--max-source-positions`](https://github.com/pytorch/fairseq/blob/master/fairseq/tasks/translation.py#L147) parameter to set the maximum sequence length.
